profile/c5g7-cmfd.cpp

```diff
@@ -0,0 +1,509 @@
+#include "../src/CPUSolver.h"
+#include "../src/Solver.h"
+#include "../src/log.h"
+#include <array>
+#include <iostream>
+
+int main(){
+
+  /* Define simulation parameters */
+  #ifdef OPENMP
+  int num_threads = omp_get_num_procs();
+  #else
+  int num_threads = 1;
+  #endif
+  double track_spacing = 0.1;
+  int num_azim = 4;
+  double tolerance = 1e-5;
+  int max_iters = 1000;
+
+  /* Set logging information */
+  set_log_level("NORMAL");
+  log_printf(TITLE, "Simulating the OECD's C5G7 Benchmark Problem...");
+ 
+  /* Define material properties */
+  log_printf(NORMAL, "Defining material properties..."); 
+ 
+  std::map<std::string, std::array<double, 7> > sigmaA;
```

Comment:
Per the Google C++ Style Guide ([link](https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Variable_Names)), as well as OpenMOC's own Style Guide ([link](https://mit-crpg.github.io/OpenMOC/devguide/style_guide.html#variable-names)), variable names should be all lowercase with words separated by underscores.